provider/ansible/example.rb

```diff
@@ -148,9 +149,16 @@ def compiled_code(code, hash)
         end
       end
 
+      def a_or_an(item_name)
+        %w[a e i o u].include?(item_name[0].downcase) ? 'an' : 'a'
```

Comment:
This is _a_ once in a lifetime function (but certainly good enough for its purpose :-)

Reply:
haha, good example! A user (not an user) was another one I found.

I'm going to just have to whitelist certain words. I'm surprised Ruby's ActiveSupport library doesn't have a function like this.